--- initng-0.6.8-orig/plugins/reload/initng_reload.c	2006-06-11 22:37:18.000000000 +0000
+++ initng-0.6.8/plugins/reload/initng_reload.c	2006-10-19 15:46:05.218211750 +0000
@@ -44,11 +44,11 @@
 
 INITNG_PLUGIN_MACRO;
 
-#define SAVE_FILE      VARDIR "/initng_db_backup.v15"
-#define SAVE_FILE_FAKE VARDIR "/initng_db_backup_fake.v15"
+#define SAVE_FILE      VARDIR "/run/initng_db_backup.v15"
+#define SAVE_FILE_FAKE VARDIR "/run/initng_db_backup_fake.v15"
 
-#define SAVE_FILE_V13      VARDIR "/initng_db_backup.v13"
-#define SAVE_FILE_FAKE_v13 VARDIR "/initng_db_backup_fake.v13"
+#define SAVE_FILE_V13      VARDIR "/run/initng_db_backup.v13"
+#define SAVE_FILE_FAKE_v13 VARDIR "/run/initng_db_backup_fake.v13"
 
 static int write_file(const char *filename);
 static int read_file(const char *filename);
