--- ./plugins/reload/initng_reload.c~	2005-11-14 06:10:58.000000000 +0200
+++ ./plugins/reload/initng_reload.c	2005-11-17 16:20:01.000000000 +0200
@@ -37,8 +37,8 @@
 
 #include "initng_reload.h"
 
-#define SAVE_FILE "/var/initng_db_backup.v9"
-#define SAVE_FILE_FAKE "/var/initng_db_backup.v9"
+#define SAVE_FILE "/var/run/initng_db_backup.v9"
+#define SAVE_FILE_FAKE "/var/run/initng_db_backup.v9"
 
 s_command FAST_RELOAD = { 'c', "hot_reload", VOID_COMMAND, NO_OPT,
     {(void *) &cmd_fast_reload},
