--- ./src/initng.h~	2005-06-18 00:55:42.000000000 +0300
+++ ./src/initng.h	2005-06-30 20:15:30.000000000 +0300
@@ -54,7 +54,9 @@
 #define INITNG_ROOT     "/etc/initng"
 #define CONTROL_PATH    "/etc/initng"
 
+#ifndef INITNG_PLUGIN_DIR
 #define INITNG_PLUGIN_DIR "/lib/initng"
+#endif
 #define STD_OUT         "/dev/vc/12"
 #define STD_OUT2        "/dev/tty12"
 #define STD_ERR         "/dev/console"
--- initng-0.1.3/Makefile~	2005-06-17 21:38:59.000000000 +0300
+++ initng-0.1.3/Makefile	2005-06-30 19:59:35.000000000 +0300
@@ -16,13 +16,13 @@
 	@echo "and written for Gentoo, and may not work with your distro."
 	install -d $(DESTDIR)/sbin
 	install -d $(DESTDIR)/etc/initng
-	install -d $(DESTDIR)/lib/initng
+	install -d $(DESTDIR)/$(LIBDIR)/initng
 	
 	for dir in $(SUBDIRS); do make -C $$dir install || exit $?; done
 	
 	
 	
-	cp -r -u -b -v --preserve=timestamps scripts ${DESTDIR}/lib/initng
+	cp -r -u -b -v --preserve=timestamps scripts ${DESTDIR}/$(LIBDIR)/initng
 	cp -r -u -b -v --preserve=timestamps initfiles/* $(DESTDIR)/etc/initng
 	@echo
 	@echo "Initng is installed. Add "init=/sbin/initng" in your bootloader configuration"
@@ -45,7 +45,7 @@
 	@echo "belong in the distribution."
 	install -d $(DESTDIR)/sbin
 	install -d $(DESTDIR)/etc/initng
-	install -d $(DESTDIR)/lib/initng
+	install -d $(DESTDIR)/$(LIBDIR)/initng
 	
 	for dir in $(SUBDIRS); do make -C $$dir link-install || exit $?; done
 	
@@ -55,7 +55,7 @@
 	ln -s -f `pwd`/initfiles/net $(DESTDIR)/etc/initng/net
 	cp -u `pwd`/initfiles/*.i $(DESTDIR)/etc/initng
 	#cp -u `pwd`/initfiles/*.runlevel $(DESTDIR)/etc/initng
-	ln -s -f `pwd`/scripts ${DESTDIR}/lib/initng/scripts
+	ln -s -f `pwd`/scripts ${DESTDIR}/$(LIBDIR)/initng/scripts
 	@echo
 	@echo "Initng is installed. Add "init=/sbin/initng" in your bootloader configuration"
 	@echo "eg. at the end of the kernel line in /boot/grub/grub.conf"
--- ./src/Makefile~	2005-06-17 21:39:22.000000000 +0300
+++ ./src/Makefile	2005-06-30 20:01:29.000000000 +0300
@@ -29,17 +29,17 @@
 
 
 install: all
-	install libinitng.so.0.0 libinitng.so.0 libinitng.so $(DESTDIR)/lib
+	install libinitng.so.0.0 libinitng.so.0 libinitng.so $(DESTDIR)/$(LIBDIR)
 	install initng $(DESTDIR)/sbin
 link-install: all
-	install -d $(DESTDIR)/lib/initng
+	install -d $(DESTDIR)/$(LIBDIR)/initng
 	install -d $(DESTDIR)/sbin
-	ln -s -f `pwd`/libinitng.so.0.0 $(DESTDIR)/lib/libinitng.so.0.0
-	ln -s -f `pwd`/libinitng.so.0 $(DESTDIR)/lib/libinitng.so.0
-	ln -s -f `pwd`/libinitng.so $(DESTDIR)/lib/libinitng.so
+	ln -s -f `pwd`/libinitng.so.0.0 $(DESTDIR)/$(LIBDIR)/libinitng.so.0.0
+	ln -s -f `pwd`/libinitng.so.0 $(DESTDIR)/$(LIBDIR)/libinitng.so.0
+	ln -s -f `pwd`/libinitng.so $(DESTDIR)/$(LIBDIR)/libinitng.so
 	ln -s -f `pwd`/initng $(DESTDIR)/sbin/initng
 uninstall:
-	rm -f $(DESTDIR)/lib/libinitng.so.0.0 $(DESTDIR)/lib/libinitng.so.0 $(DESTDIR)/lib/libinitng.so $(DESTDIR)/sbin/initng
+	rm -f $(DESTDIR)/$(LIBDIR)/libinitng.so.0.0 $(DESTDIR)/$(LIBDIR)/libinitng.so.0 $(DESTDIR)/$(LIBDIR)/libinitng.so $(DESTDIR)/sbin/initng
 
 
 initng_toolbox.o: initng_toolbox.c initng.h initng_global.h
--- ./plugins/up/Makefile~	2005-06-17 21:39:09.000000000 +0300
+++ ./plugins/up/Makefile	2005-06-30 20:01:49.000000000 +0300
@@ -13,9 +13,9 @@
 
 
 install: libup.so.0.0
-	install libup.so.0.0 $(DESTDIR)/lib/initng
+	install libup.so.0.0 $(DESTDIR)/$(LIBDIR)/initng
 link-install: libup.so.0.0
-	install -d $(DESTDIR)/lib/initng
+	install -d $(DESTDIR)/$(LIBDIR)/initng
 	ln -s -f `pwd`/libup.so.0.0 $(DESTDIR)/lib/initng/libup.so.0.0
 uninstall:
 	rm -f $(DESTDIR)/lib/initng/libup.so.0.0
--- ./plugins/ngc/Makefile~	2005-06-17 21:39:18.000000000 +0300
+++ ./plugins/ngc/Makefile	2005-06-30 20:02:18.000000000 +0300
@@ -22,7 +22,7 @@
 install-ngc1:
 	install ngc ngdc $(DESTDIR)/sbin
 install: libngc.so.0.0
-	install libngc.so.0.0 $(DESTDIR)/lib/initng
+	install libngc.so.0.0 $(DESTDIR)/$(LIBDIR)/initng
 	mkfifo -m 600 $(DESTDIR)/etc/initng/in $(DESTDIR)/etc/initng/out $(DESTDIR)/etc/initng/in-test $(DESTDIR)/etc/initng/out-test ; /bin/true
 link-install-ngc1:
 	install -d $(DESTDIR)/sbin
@@ -33,10 +33,10 @@
 	install -d $(DESTDIR)/sbin
 	ln -s -f `pwd`/pidfilehack $(DESTDIR)/sbin/pidfilehack
 	ln -s -f `pwd`/pidfiledhack $(DESTDIR)/sbin/pidfiledhack
-	install -d $(DESTDIR)/lib/initng
-	ln -s -f `pwd`/libngc.so.0.0 $(DESTDIR)/lib/initng/libngc.so.0.0
+	install -d $(DESTDIR)/$(LIBDIR)/initng
+	ln -s -f `pwd`/libngc.so.0.0 $(DESTDIR)/$(LIBDIR)/initng/libngc.so.0.0
 uninstall:
-	rm -f $(DESTDIR)/lib/initng/libngc.so.0.0
+	rm -f $(DESTDIR)/$(LIBDIR)/initng/libngc.so.0.0
 	rm -f $(DESTDIR)/sbin/ngc
 	rm -f $(DESTDIR)/sbin/ngdc
 	rm -f $(DESTDIR)/sbin/pidfilehack $(DESTDIR)/sbin/pidfiledhack
--- ./plugins/ngc2/Makefile~	2005-06-17 21:39:10.000000000 +0300
+++ ./plugins/ngc2/Makefile	2005-06-30 20:02:40.000000000 +0300
@@ -18,17 +18,17 @@
 
 install: all
 	install ngc ngdc $(DESTDIR)/sbin
-	install libngc2.so.0.0 $(DESTDIR)/lib/initng
+	install libngc2.so.0.0 $(DESTDIR)/$(LIBDIR)/initng
 link-install: all
 	install -d $(DESTDIR)/sbin
 	ln -s -f `pwd`/ngc $(DESTDIR)/sbin/ngc
 	ln -s -f $(DESTDIR)/sbin/ngc $(DESTDIR)/sbin/ngdc
-	install -d $(DESTDIR)/lib/initng
-	ln -s -f `pwd`/libngc2.so.0.0 $(DESTDIR)/lib/initng/libngc2.so.0.0
+	install -d $(DESTDIR)/$(LIBDIR)/initng
+	ln -s -f `pwd`/libngc2.so.0.0 $(DESTDIR)/$(LIBDIR)/initng/libngc2.so.0.0
 uninstall:
 	rm -f $(DESTDIR)/sbin/ngc
 	rm -f $(DESTDIR)/sbin/ngdc
-	rm -f $(DESTDIR)/lib/initng/libngc2.so.0.0
+	rm -f $(DESTDIR)/$(LIBDIR)/initng/libngc2.so.0.0
 
 clean:
 	rm *.o libngc2.so.0.0 2>&1 ; /bin/true
--- ./plugins/bash_launcher/Makefile~	2005-06-17 21:39:10.000000000 +0300
+++ ./plugins/bash_launcher/Makefile	2005-06-30 20:03:03.000000000 +0300
@@ -12,12 +12,12 @@
 
 
 install: libbashlaunch.so.0.0
-	install libbashlaunch.so.0.0 $(DESTDIR)/lib/initng
+	install libbashlaunch.so.0.0 $(DESTDIR)/$(LIBDIR)/initng
 link-install: libbashlaunch.so.0.0
-	install -d $(DESTDIR)/lib/initng
-	ln -s -f `pwd`/libbashlaunch.so.0.0 $(DESTDIR)/lib/initng/libbashlaunch.so.0.0
+	install -d $(DESTDIR)/$(LIBDIR)/initng
+	ln -s -f `pwd`/libbashlaunch.so.0.0 $(DESTDIR)/$(LIBDIR)/initng/libbashlaunch.so.0.0
 uninstall:
-	rm -f $(DESTDIR)/lib/initng/libbashlaunch.so.0.0
+	rm -f $(DESTDIR)/$(LIBDIR)/initng/libbashlaunch.so.0.0
 
 clean:
-	rm *.o libbashlaunch.so.0.0 ; /bin/true
\ No newline at end of file
+	rm *.o libbashlaunch.so.0.0 ; /bin/true
--- ./plugins/rlparser/Makefile~	2005-06-17 21:39:10.000000000 +0300
+++ ./plugins/rlparser/Makefile	2005-06-30 20:03:24.000000000 +0300
@@ -13,12 +13,12 @@
 
 
 install: librlparser.so.0.0
-	install librlparser.so.0.0 $(DESTDIR)/lib/initng
+	install librlparser.so.0.0 $(DESTDIR)/$(LIBDIR)/initng
 link-install: librlparser.so.0.0
-	install -d $(DESTDIR)/lib/initng
-	ln -s -f `pwd`/librlparser.so.0.0 $(DESTDIR)/lib/initng/librlparser.so.0.0
+	install -d $(DESTDIR)/$(LIBDIR)/initng
+	ln -s -f `pwd`/librlparser.so.0.0 $(DESTDIR)/$(LIBDIR)/initng/librlparser.so.0.0
 uninstall:
-	rm -f $(DESTDIR)/lib/initng/librlparser.so.0.0
+	rm -f $(DESTDIR)/$(LIBDIR)/initng/librlparser.so.0.0
 
 clean:
 	rm *.o librlparser.so.0.0 ; /bin/true
--- ./plugins/cpout/Makefile~	2005-06-17 21:39:10.000000000 +0300
+++ ./plugins/cpout/Makefile	2005-06-30 20:03:47.000000000 +0300
@@ -11,12 +11,12 @@
 
 
 install: libcpout.so.0.0
-	install libcpout.so.0.0 $(DESTDIR)/lib/initng
+	install libcpout.so.0.0 $(DESTDIR)/$(LIBDIR)/initng
 link-install: libcpout.so.0.0
-	install -d $(DESTDIR)/lib/initng
-	ln -s -f `pwd`/libcpout.so.0.0 $(DESTDIR)/lib/initng/libcpout.so.0.0
+	install -d $(DESTDIR)/$(LIBDIR)/initng
+	ln -s -f `pwd`/libcpout.so.0.0 $(DESTDIR)/$(LIBDIR)/initng/libcpout.so.0.0
 uninstall:
-	rm -f $(DESTDIR)/lib/initng/libcpout.so.0.0
+	rm -f $(DESTDIR)/$(LIBDIR)/initng/libcpout.so.0.0
 
 clean:
-	rm *.o libcpout.so.0.0 ; /bin/true
\ No newline at end of file
+	rm *.o libcpout.so.0.0 ; /bin/true
--- ./plugins/ezxmlparser/Makefile~	2005-06-17 21:39:18.000000000 +0300
+++ ./plugins/ezxmlparser/Makefile	2005-06-30 20:04:07.000000000 +0300
@@ -9,12 +9,12 @@
 
 
 install: libezxmlparser.so.0.0
-	install libezxmlparser.so.0.0 $(DESTDIR)/lib/initng
+	install libezxmlparser.so.0.0 $(DESTDIR)/$(LIBDIR)/initng
 link-install: libezxmlparser.so.0.0
-	install -d $(DESTDIR)/lib/initng
-	ln -s -f `pwd`/libezxmlparser.so.0.0 $(DESTDIR)/lib/initng/libezxmlparser.so.0.0
+	install -d $(DESTDIR)/$(LIBDIR)/initng
+	ln -s -f `pwd`/libezxmlparser.so.0.0 $(DESTDIR)/$(LIBDIR)/initng/libezxmlparser.so.0.0
 uninstall:
-	rm -f $(DESTDIR)/lib/initng/libezxmlparser.so.0.0
+	rm -f $(DESTDIR)/$(LIBDIR)/initng/libezxmlparser.so.0.0
 
 
 ezxml.o: ezxml.c ezxml.h
--- ./plugins/simple_launcher/Makefile~	2005-06-17 21:39:18.000000000 +0300
+++ ./plugins/simple_launcher/Makefile	2005-06-30 20:04:28.000000000 +0300
@@ -11,12 +11,12 @@
 
 
 install: libslaunch.so.0.0
-	install libslaunch.so.0.0 $(DESTDIR)/lib/initng
+	install libslaunch.so.0.0 $(DESTDIR)/$(LIBDIR)/initng
 link-install: libslaunch.so.0.0
-	install -d $(DESTDIR)/lib/initng
-	ln -s -f `pwd`/libslaunch.so.0.0 $(DESTDIR)/lib/initng/libslaunch.so.0.0
+	install -d $(DESTDIR)/$(LIBDIR)/initng
+	ln -s -f `pwd`/libslaunch.so.0.0 $(DESTDIR)/$(LIBDIR)/initng/libslaunch.so.0.0
 uninstall:
-	rm -f $(DESTDIR)/lib/initng/libslaunch.so.0.0
+	rm -f $(DESTDIR)/$(LIBDIR)/initng/libslaunch.so.0.0
 
 clean:
-	rm *.o libslaunch.so.0.0 ; /bin/true
\ No newline at end of file
+	rm *.o libslaunch.so.0.0 ; /bin/true
--- ./plugins/dash_launcher/Makefile~	2005-06-17 21:39:17.000000000 +0300
+++ ./plugins/dash_launcher/Makefile	2005-06-30 20:05:22.000000000 +0300
@@ -17,12 +17,12 @@
 
 
 install: libdashlaunch.so.0.0
-	install libdashlaunch.so.0.0 $(DESTDIR)/lib/initng
+	install libdashlaunch.so.0.0 $(DESTDIR)/$(LIBDIR)/initng
 link-install: libdashlaunch.so.0.0
-	install -d $(DESTDIR)/lib/initng
-	ln -s -f `pwd`/libdashlaunch.so.0.0 $(DESTDIR)/lib/initng/libdashlaunch.so.0.0
+	install -d $(DESTDIR)/$(LIBDIR)/initng
+	ln -s -f `pwd`/libdashlaunch.so.0.0 $(DESTDIR)/$(LIBDIR)/initng/libdashlaunch.so.0.0
 uninstall:
-	rm -f $(DESTDIR)/lib/initng/libdashlaunch.so.0.0
+	rm -f $(DESTDIR)/$(LIBDIR)/initng/libdashlaunch.so.0.0
 
 clean:
 	rm *.o libdashlaunch.so.0.0 2>/dev/null ; /bin/true
--- ./plugins/stdout/Makefile~	2005-06-17 21:39:09.000000000 +0300
+++ ./plugins/stdout/Makefile	2005-06-30 20:05:45.000000000 +0300
@@ -13,12 +13,12 @@
 
 
 install: libstdout.so.0.0
-	install libstdout.so.0.0 $(DESTDIR)/lib/initng
+	install libstdout.so.0.0 $(DESTDIR)/$(LIBDIR)/initng
 link-install: libstdout.so.0.0
-	install -d $(DESTDIR)/lib/initng
-	ln -s -f `pwd`/libstdout.so.0.0 $(DESTDIR)/lib/initng/libstdout.so.0.0
+	install -d $(DESTDIR)/$(LIBDIR)/initng
+	ln -s -f `pwd`/libstdout.so.0.0 $(DESTDIR)/$(LIBDIR)/initng/libstdout.so.0.0
 uninstall:
-	rm -f $(DESTDIR)/lib/initng/libstdout.so.0.0
+	rm -f $(DESTDIR)/$(LIBDIR)/initng/libstdout.so.0.0
 
 
 clean:
--- ./plugins/xmlconfig/Makefile~	2005-06-17 21:39:09.000000000 +0300
+++ ./plugins/xmlconfig/Makefile	2005-06-30 20:06:06.000000000 +0300
@@ -9,12 +9,12 @@
 
 
 install: libxmlconfig.so.0.0
-	install libxmlconfig.so.0.0 $(DESTDIR)/lib/initng
+	install libxmlconfig.so.0.0 $(DESTDIR)/$(LIBDIR)/initng
 link-install: libxmlconfig.so.0.0
-	install -d $(DESTDIR)/lib/initng
-	ln -s -f `pwd`/libxmlconfig.so.0.0 $(DESTDIR)/lib/initng/libxmlconfig.so.0.0
+	install -d $(DESTDIR)/$(LIBDIR)/initng
+	ln -s -f `pwd`/libxmlconfig.so.0.0 $(DESTDIR)/$(LIBDIR)/initng/libxmlconfig.so.0.0
 uninstall:
-	rm -f $(DESTDIR)/lib/initng/libxmlconfig.so.0.0
+	rm -f $(DESTDIR)/$(LIBDIR)/initng/libxmlconfig.so.0.0
 
 
 ezxml.o: ezxml.c ezxml.h
--- ./plugins/splashout/Makefile~	2005-06-17 21:39:09.000000000 +0300
+++ ./plugins/splashout/Makefile	2005-06-30 20:06:31.000000000 +0300
@@ -18,14 +18,14 @@
 
 
 install: libsplashout.so.0.0
-	install libsplashout.so.0.0 $(DESTDIR)/lib/initng
+	install libsplashout.so.0.0 $(DESTDIR)/$(LIBDIR)/initng
 	for dir in $(SUBDIRS); do make -C $$dir $@ || exit $?; done
 link-install: libsplashout.so.0.0
-	install -d $(DESTDIR)/lib/initng
-	ln -s -f `pwd`/libsplashout.so.0.0 $(DESTDIR)/lib/initng/libsplashout.so.0.0
+	install -d $(DESTDIR)/$(LIBDIR)/initng
+	ln -s -f `pwd`/libsplashout.so.0.0 $(DESTDIR)/$(LIBDIR)/initng/libsplashout.so.0.0
 	for dir in $(SUBDIRS); do make -C $$dir $@ || exit $?; done
 uninstall:
-	rm -f $(DESTDIR)/lib/initng/libsplashout.so.0.0
+	rm -f $(DESTDIR)/$(LIBDIR)/initng/libsplashout.so.0.0
 	for dir in $(SUBDIRS); do make -C $$dir $@ || exit $?; done
 
 
--- ./plugins/rcparser/Makefile~	2005-06-17 21:39:09.000000000 +0300
+++ ./plugins/rcparser/Makefile	2005-06-30 20:06:55.000000000 +0300
@@ -15,12 +15,12 @@
 
 
 install: librcparser.so.0.0
-	install librcparser.so.0.0 $(DESTDIR)/lib/initng
+	install librcparser.so.0.0 $(DESTDIR)/$(LIBDIR)/initng
 link-install: librcparser.so.0.0
-	install -d $(DESTDIR)/lib/initng
-	ln -s -f `pwd`/librcparser.so.0.0 $(DESTDIR)/lib/initng/librcparser.so.0.0
+	install -d $(DESTDIR)/$(LIBDIR)/initng
+	ln -s -f `pwd`/librcparser.so.0.0 $(DESTDIR)/$(LIBDIR)/initng/librcparser.so.0.0
 uninstall:
-	rm -f $(DESTDIR)/lib/initng/librcparser.so.0.0
+	rm -f $(DESTDIR)/$(LIBDIR)/initng/librcparser.so.0.0
 
 clean:
 	rm *.o librcparser.so.0.0 ; /bin/true
--- ./plugins/initctl/Makefile~	2005-06-17 21:39:17.000000000 +0300
+++ ./plugins/initctl/Makefile	2005-06-30 20:07:12.000000000 +0300
@@ -13,12 +13,12 @@
 
 
 install: libinitctl.so.0.0
-	install libinitctl.so.0.0 $(DESTDIR)/lib/initng
+	install libinitctl.so.0.0 $(DESTDIR)/$(LIBDIR)/initng
 link-install: libinitctl.so.0.0
-	install -d $(DESTDIR)/lib/initng
-	ln -s -f `pwd`/libinitctl.so.0.0 $(DESTDIR)/lib/initng/libinitctl.so.0.0
+	install -d $(DESTDIR)/$(LIBDIR)/initng
+	ln -s -f `pwd`/libinitctl.so.0.0 $(DESTDIR)/$(LIBDIR)/initng/libinitctl.so.0.0
 uninstall:
-	rm -f $(DESTDIR)/lib/initng/libinitctl.so.0.0
+	rm -f $(DESTDIR)/$(LIBDIR)/initng/libinitctl.so.0.0
 
 clean:
 	rm *.o libinitctl.so.0.0 ; /bin/true
--- ./plugins/iparser/Makefile~	2005-06-17 21:39:17.000000000 +0300
+++ ./plugins/iparser/Makefile	2005-06-30 20:07:30.000000000 +0300
@@ -13,12 +13,12 @@
 
 
 install: libiparser.so.0.0
-	install libiparser.so.0.0 $(DESTDIR)/lib/initng
+	install libiparser.so.0.0 $(DESTDIR)/$(LIBDIR)/initng
 link-install: libiparser.so.0.0
-	install -d $(DESTDIR)/lib/initng
-	ln -s -f `pwd`/libiparser.so.0.0 $(DESTDIR)/lib/initng/libiparser.so.0.0
+	install -d $(DESTDIR)/$(LIBDIR)/initng
+	ln -s -f `pwd`/libiparser.so.0.0 $(DESTDIR)/$(LIBDIR)/initng/libiparser.so.0.0
 uninstall:
-	rm -f $(DESTDIR)/lib/initng/libiparser.so.0.0
+	rm -f $(DESTDIR)/$(LIBDIR)/initng/libiparser.so.0.0
 
 
 clean:
--- ./plugins/readahead/Makefile~	2005-06-17 21:39:10.000000000 +0300
+++ ./plugins/readahead/Makefile	2005-06-30 20:07:48.000000000 +0300
@@ -11,14 +11,14 @@
 
 
 install: libreadahead.so.0.0
-	install libreadahead.so.0.0 $(DESTDIR)/lib/initng
+	install libreadahead.so.0.0 $(DESTDIR)/$(LIBDIR)/initng
 	mkdir -p $(DESTDIR)/etc/initng/readahead.d/
 	sh scripts/makeall
 link-install: libreadahead.so.0.0
-	install -d $(DESTDIR)/lib/initng
-	ln -s -f `pwd`/libreadahead.so.0.0 $(DESTDIR)/lib/initng/libreadahead.so.0.0
+	install -d $(DESTDIR)/$(LIBDIR)/initng
+	ln -s -f `pwd`/libreadahead.so.0.0 $(DESTDIR)/$(LIBDIR)/initng/libreadahead.so.0.0
 uninstall:
-	rm -f $(DESTDIR)/lib/initng/libreadahead.so.0.0
+	rm -f $(DESTDIR)/$(LIBDIR)/initng/libreadahead.so.0.0
 
 
 clean:
--- ./plugins/up/Makefile~	2005-06-30 20:20:04.000000000 +0300
+++ ./plugins/up/Makefile	2005-06-30 20:27:10.000000000 +0300
@@ -18,7 +18,7 @@
 	install -d $(DESTDIR)/$(LIBDIR)/initng
 	ln -s -f `pwd`/libup.so.0.0 $(DESTDIR)/lib/initng/libup.so.0.0
 uninstall:
-	rm -f $(DESTDIR)/lib/initng/libup.so.0.0
+	rm -f $(DESTDIR)/$(LIBDIR)/initng/libup.so.0.0
 
 
 clean:
