--- ./plugins/lockfile/initng_lockfile.c~	2006-04-13 01:19:43.000000000 +0300
+++ ./plugins/lockfile/initng_lockfile.c	2006-04-20 12:33:26.187989468 +0300
@@ -56,9 +56,12 @@
 		// service states from initng_is.h
 		if (IS_UP(service))
 		{
+			int fd;
 			D_("service got up\n");
-			creat(lockfile, S_IREAD);
-
+			fd = creat(lockfile, 0640);
+			if (fd != -1) {
+				close(fd);
+			}
 		}
 		else if (IS_DOWN(service))
 		{
