diff -ur initng-0.5.4/Makefile.am initng-0.5.4.new/Makefile.am
--- initng-0.5.4/Makefile.am	2006-02-22 13:12:24.000000000 +0200
+++ initng-0.5.4.new/Makefile.am	2006-03-08 01:54:43.000000000 +0200
@@ -1,6 +1,4 @@
-SUBDIRS = tools src plugins scripts doc fixes initfiles gui devtool
-
-sbin_SCRIPTS = gen_system_runlevel
+SUBDIRS = tools src plugins doc gui devtool
 
 BUILT_SOURCES = initng-paths.h
 CLEANFILES =    initng-paths.h
@@ -21,7 +19,6 @@
 	@echo "eg. at the end of the kernel line in /boot/grub/grub.conf"
 	@echo "Have a lot of fun!"
 	@echo
-	$(top_srcdir)/gen_system_runlevel $(DESTDIR)
 	@echo
 	@echo "If you have initng running, you can reload it with:"
 	@echo "# /sbin/ngc -c"
diff -ur initng-0.5.4/doc/Makefile.am initng-0.5.4.new/doc/Makefile.am
--- initng-0.5.4/doc/Makefile.am	2006-02-22 13:12:29.000000000 +0200
+++ initng-0.5.4.new/doc/Makefile.am	2006-03-08 01:54:43.000000000 +0200
@@ -1,7 +1,5 @@
 man_MANS = \
-	gen_system_runlevel.8 \
 	initng.8 \
-	install_service.8 \
 	ngc.8 \
 	ngdc.8 \
 	ng-update.8 \
diff -ur initng-0.5.4/tools/Makefile.am initng-0.5.4.new/tools/Makefile.am
--- initng-0.5.4/tools/Makefile.am	2006-02-22 13:12:29.000000000 +0200
+++ initng-0.5.4.new/tools/Makefile.am	2006-03-08 01:54:43.000000000 +0200
@@ -2,12 +2,10 @@
 
 AUTOMAKE_OPTIONS = gnu
 
-install_service_SOURCES = install_service.c
 system_off_SOURCES = system_off.c
-killalli5_SOURCES = killall5.c
 
-sbin_PROGRAMS = system_off install_service killalli5
-sbin_SCRIPTS  = ng-update shutdown_script initng-segfault
+sbin_PROGRAMS = system_off
+sbin_SCRIPTS  = ng-update initng-segfault
 
 
 if INSTALL_AS_INIT
